scripts/hv_stats/Jhv_movies/histogram.html

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
Is this autogenerated? Where is it getting its data from?

Reply:
Yes, it's autogenerated by hv_stats/hv_stats.py. It queries our databases and uses the python package bokeh to autogenerate plots. The subdirectories coverages, embed_service, histograms, hv_movies,hv_screenshots, hv_student, Jhv_movies and service_usage are all autogenerated and should not be part of the repo. The script could stand to be a little better, for example, the log files should go in a proper log directory. I'd hope there is a better solution for such autogenerated content.

Reply:
I think it's okay that they're here. Auto-generated files should be in .gitignore though so I'll add them to that.